typescript/src/purchasesConfiguration.ts

```diff
@@ -17,13 +17,6 @@ export interface PurchasesConfiguration {
    * A unique id for identifying the user
    */
   appUserID?: string | null;
-  /**
-   * An optional boolean. Set this to TRUE if you have your own IAP implementation and
-   * want to use only RevenueCat's backend. Default is FALSE. If you are on Android and setting this to ON, you will have
-   * to acknowledge the purchases yourself.
-   * @deprecated Use purchasesAreCompletedBy instead.
-   */
-  observerMode?: boolean;
```

Comment:
It’s probably fine but this is a breaking change so I would label this PR as breaking. We believe it’s ok to make many breaking changes in PHC as long as we don’t propagate them to the hybrids